client/asset/btc/btc.go

```diff
@@ -42,24 +41,68 @@ const (
 	// rpcclient.Client's GetBlockVerboseTx appears to be busted.
 	methodGetBlockVerboseTx = "getblock"
 	methodGetNetworkInfo    = "getnetworkinfo"
-	BipID                   = 0
+	// BipID is the BIP-0044 asset ID.
+	BipID = 0
 	// The default fee is passed to the user as part of the asset.WalletInfo
 	// structure.
-	defaultFee         = 112
+	defaultFee         = 100
 	minNetworkVersion  = 190000
 	minProtocolVersion = 70015
+	// splitTxBaggage is the total number of additional bytes associated with
+	// using a split transaction to fund a swap.
+	splitTxBaggage = dexbtc.MimimumTxOverhead + 1 + dexbtc.RedeemP2WPKHInputSize + 2*(1+dexbtc.P2PKHOutputSize)
 )
 
 var (
 	// blockTicker is the delay between calls to check for new blocks.
-	blockTicker = time.Second
+	blockTicker    = time.Second
+	fallbackFeeKey = "fallbackfee"
+	configOpts     = []*asset.ConfigOption{
+		{
+			Key:         "walletname",
+			DisplayName: "Wallet Name",
+			Description: "The wallet name",
+		},
+		{
+			Key:         "rpcuser",
+			DisplayName: "JSON-RPC Username",
+			Description: "Bitcoin's 'rpcuser' setting",
+		},
+		{
+			Key:         "rpcpassword",
+			DisplayName: "JSON-RPC Password",
+			Description: "Bitcoin's 'rpcpassword' setting",
+			NoEcho:      true,
+		},
+		{
+			Key:         "rpcbind",
+			DisplayName: "JSON-RPC Address",
+			Description: "<addr> or <addr>:<port> (default 'localhost')",
+		},
+		{
+			Key:         "rpcport",
+			DisplayName: "JSON-RPC Port",
+			Description: "Port for RPC connections (if not set in Address)",
+		},
+		{
+			Key:          fallbackFeeKey,
+			DisplayName:  "Fallback fee rate",
+			Description:  "Bitcoin's 'fallbackfee' rate. Units: Sats/kB",
+			DefaultValue: defaultFee,
+		},
+		{
+			Key:         "txsplit",
+			DisplayName: "Pre-split funding inputs",
+			Description: "Pre-split funding inputs to prevent locking funds into an order for which a change output may not be immediately available. Only used for standing-type orders.",
```

Comment:
Maybe "to prevent locking *excess* funds".

I get that "change output may not be immediately available" refers to when the swap contract txn happens shortly after the order placement, thus making the change available, but it's unlikely most users will get that.  Plus, even if they did get that, they should understand that a swap could take hours.  How about something more accessible to average users like `When placing an order, create a "split" transaction to fund the order without locking more of the wallet balance than necessary.  Otherwise, excess funds may be reserved to fund the order until the swap contract transaction is created during trade settlement or the order is canceled.  This an extra transaction for which network mining fees are paid.  Most useful for standing-type orders that may remain on the book indefinitely.`

Also, it currently seems to be used for all trade orders, not just standing.

Reply:
> Also, it currently seems to be used for all trade orders, not just standing.

Oh yeah. I had a solution working for this, but backed off for some reason. I'll get it in now.

Reply:
So adding asset-specific order options is probably doable, but a large undertaking. For now, I'm just informing the `ExchangeWallet` whether the funds are for a standing or immediate order, and the funds aren't being split if it's immediate. See aa5fc57.

Reply:
Works for me.